pom.xml

```diff
@@ -15,9 +15,18 @@
 
     <!-- Maven plugin versions -->
     <maven.compiler.version>3.10.1</maven.compiler.version>
+
+    <!-- Project dependency version -->
+    <flink.version>1.19.0</flink.version>
```

Comment:
Could be good to use `1.19.1` and extract it into a property covering this and the confluent registry format version